VideoRental/Video.java

```diff
@@ -0,0 +1,73 @@
+import java.util.Date;
+
+public class Video {
+	private String title ;
+
+	private int priceCode ;
+	public static final int REGULAR = 1 ;
+	public static final int NEW_RELEASE =2 ;
+	
+	private int videoType ;
+	public static final int VHS = 1 ;
+	public static final int CD = 2 ;
+	public static final int DVD = 3 ;
+	
+	private Date registeredDate ;
+	private boolean rented ;
+	
+	public Video(String title, int videoType, int priceCode, Date registeredDate) {
+		this.setTitle(title) ;
+		this.setVideoType(videoType) ;
+		this.setPriceCode(priceCode) ;
+		this.registeredDate = registeredDate ;
+	}
```

Comment:
I think if you add "rented" for avoid the "rental" duplication of "video", better than now.
Of course, if you fix, you have to fix below too.

VideoRental/Rental.java

	public Rental(Video video) {
               if (video.rented == true) return ;
		this.video = video ;
		status = 0 ;
		rentDate = new Date() ;
	}

Reply:
That is good idea. I agree.